axelrod/strategy_transformers.py

```diff
@@ -228,11 +297,21 @@ def dual_wrapper(player, opponent, proposed_action):
     -------
     action: an axelrod.Action, C or D
     """
+
     if not player.history:
-        player.original_player = player.original_class(**player.init_kwargs)
+        player.use_history = []
```

Comment:
> Does this work, testing-wise?

Yes I think it does. @theref can you confirm? 

I expect we would get more failures for more than 10 turns (especially for the complex FSMs) but in the interest of run time I think this is a sufficient test.

> after i added a function to switch player.cooperations and player.defections (and then switch them back) they all passed.

The trick will be to get an automatic function that doesn't just switch `player.cooperation` and `.defections` but every and any attribute that can affect a strategy (take a look at the player equality method, we essentially go in and grab relevant attributes there).